doc/cygnus-ngsi/installation_and_administration_guide/configuration_examples.md

```diff
@@ -1,47 +1,470 @@
+
+# sources
+cygnus-ngsi.sources.src-all.type = http
+cygnus-ngsi.sources.src-all.channels = ch-client-1 ch-client-2 ... ch-client-N ch-def
+cygnus-ngsi.sources.src-all.selector.type = multiplexing
+cygnus-ngsi.sources.src-all.selector.header = fiware-service
+cygnus-ngsi.sources.src-all.selector.mappings.<service1> = ch-client-1
+cygnus-ngsi.sources.src-all.selector.mappings.<service2> = ch-client-2
+...
+cygnus-ngsi.sources.src-all.selector.mappings.<serviceN> = ch-client-N
+cygnus-ngsi.sources.src-all.selector.mappings.default = ch-def
+cygnus-ngsi.sources.src-all.port = 5050
+
+cygnus-ngsi.sources.src-all...
+
+# sinks
+cygnus-ngsi.sinks.sink-client-1.type = ...
+cygnus-ngsi.sinks.sink-client-1.channel = ch-client-1
+cygnus-ngsi.sinks.sink-client-1...
+
+cygnus-ngsi.sinks.sink-client-2.type = ...
+cygnus-ngsi.sinks.sink-client-2.channel = ch-client-2
+cygnus-ngsi.sinks.sink-client-2...
+
+...
+
+cygnus-ngsi.sinks.sink-client-N.type = ...
+cygnus-ngsi.sinks.sink-client-N.channel = ch-client-N
+cygnus-ngsi.sinks.sink-client-N...
+
+cygnus-ngsi.sinks.sink-def.type = ...
+cygnus-ngsi.sinks.sink-def.channel = ch-def
+cygnus-ngsi.sinks.sink-def...
+
+# channels
+cygnus-ngsi.channels.ch-client-1.type = ...
+cygnus-ngsi.channels.ch-client-1...
+
+cygnus-ngsi.channels.ch-client-2.type = ...
+cygnus-ngsi.channels.ch-client-2...
+
+...
+
+cygnus-ngsi.channels.ch-client-N.type = ...
+cygnus-ngsi.channels.ch-client-N...
+
+cygnus-ngsi.channels.ch-def.type = ...
+cygnus-ngsi.channels.ch-def...
+```
+
+The related `cygnus_instance_all.conf` should look like:
+
+```
+$ cat /usr/cygnus/conf/cygnus_instance_all.conf
+CYGNUS_USER=cygnus
+CONFIG_FOLDER=/usr/cygnus/conf
+CONFIG_FILE=/usr/cygnus/conf/agent_all.conf
+AGENT_NAME=cygnus-ngsi
+LOGFILE_NAME=cygnus.log
+ADMIN_PORT=8081
+POLLING_INTERVAL=30
+```
+
+Use this kind of advanced configuration in the most restrictive scenarios, where both hardware and exposed ports are very limited and/or highly valuable.
 
 [Top](#top)
 
-##<a name="section4"></a>Single source, multiple storages, parallel sinking
-To be done
+#### <a name="section2.1.4"></a>Orion Context Broker subscriptions
+It seems obvious a per client subscription is required when a per client listener is configured for Cygnus (either by using multiple agents, either by using a single agent with multiple listeners). In that case, each subscription will point to a different endpoint URL.
+
+Nevertheless, it may not seem so obvious how subscriptions look like when a single Cygnus agent with a single listener internally multiplexes the notified context data based on the FIWARE service. In that case, a suscription per client is also required (since Orion does not allow to subscribe to all FIWARE services, due to security reasons), but all subscriptions must point to the same single endpoint URL.
 
 [Top](#top)
 
-##<a name="section5"></a>Multiple sources 
-To be done
+### <a name="section2.2"></a>Scenarios involving multiple FIWARE services
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1420/commits/ea511e5c58c65fc69304692013f33e82ae986f1a